src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -581,3 +586,21 @@ float VtolType::getOpenLoopFrontTransitionTime() const
 {
 	return getFrontTransitionTimeFactor() * _param_vt_f_tr_ol_tm.get();
 }
+float VtolType::getTransitionAirspeed() const
+{
+	return  math::max(_param_vt_arsp_trans.get(), getMinimumAirspeed());
+}
+float VtolType::getMinimumAirspeed() const
```

Comment:
Can you raname it to MinimumTransitionAirspeed()? As otherwise it may be confused with FW_AIRSPD_MIN, which has nothing to do with the speed here.

Reply:
Done